app/client/components/delivery/address/deliveryAddressEditConfirmation.tsx

```diff
@@ -1,115 +1,318 @@
+              `}
+            >
+              I understand that this address change will affect the following
+              subscriptions
+            </p>
+            <ProductDescriptionListTable
+              content={addressChangedInformationContext}
+              seperateEachRow
+            />
+            <LinkButton
+              css={css`
+                margin-top: ${space[3]}px;
+                ${minWidth.tablet} {
+                  margin-top: ${space[5]}px;
+                }
+              `}
+              href={navLinks.subscriptions.link}
+            >
+              Go back to Subscriptions
+            </LinkButton>
+            <p
+              css={css`
+                ${textSans.medium()};
+                margin-top: ${space[12]}px;
+                color: ${palette.neutral[46]};
+              `}
+            >
+              If you need seperate delivery addresses for each of your
+              subscriptions, please{" "}
+              <span
+                css={css`
+                  cursor: pointer;
+                  color: ${palette.brand[500]};
+                  text-decoration: underline;
+                `}
+                onClick={() =>
+                  setTopCallCentreNumbersVisibility(!showTopCallCentreNumbers)
+                }
+              >
+                contact us
+              </span>
+              .
+            </p>
+            {showTopCallCentreNumbers && <CallCentreEmailAndNumbers />}
+          </PageNavAndContentContainer>
+        </>
       ) : (
         visuallyNavigateToParent(props)
       )}
     </WizardStep>
   );
 };
 
-interface AddressDisplayContainerProps {
-  title: string;
-  children: ReactElement<any> | null;
-}
-const AddressDisplayContainer = (props: AddressDisplayContainerProps) => (
-  <section
-    css={css`
-      border: 1px solid ${palette.neutral["86"]};
-      padding: 48px 0 14px;
-      position: relative;
-      section {
-        margin: 14px 0;
-        padding: 0 14px;
-        position: relative;
-      }
-      section + section {
-        padding-top: 14px;
-        border-top: 1px solid ${palette.neutral["86"]};
-      }
-    `}
-  >
-    <h1
-      css={css`
-        width: 100%;
-        position: absolute;
-        top: 0;
-        left: 0;
-        margin: 0;
-        padding: 0 14px;
-        ${textSans.medium()};
-        font-weight: bold;
-        line-height: 48px;
-        background-color: ${palette.neutral["97"]};
-        border-bottom: 1px solid ${palette.neutral["86"]};
-      `}
-    >
-      New address details
-    </h1>
-    <>{props.children}</>
-  </section>
-);
+export const DeliveryAddressEditConfirmation = (props: RouteableStepProps) => {
+  const addressContext = useContext(NewDeliveryAddressContext);
+  const contactIdContext = useContext(ContactIdContext);
+  const addressChangedInformationContext = useContext(
+    AddressChangedInformationContext
+  );
+
+  const addressChangeEmailCopy = [
```

Comment:
although we know this is currently used for the email copy, it will have other uses so would rather the name was more like 
```suggestion
  const addressChangeInformationCopy = [
```